package.json

```diff
@@ -22,6 +22,7 @@
     "express": "4.15.3",
     "express-graphql": "0.6.7",
     "graphql": "0.10.5",
+    "node-fetch": "^1.7.1",
```

Comment:
@Sanderson239 please remove the `^` from this packages version number to lock it into place, thanks!